docs/language/internals.rst

```diff
@@ -50,131 +50,131 @@ respectively, to ``False``.
 
 .. _hysequence:
 
-HySequence
+Sequence
 ~~~~~~~~~~
 
-``hy.models.HySequence`` is the abstract base class of "iterable" Hy
-models, such as HyExpression and HyList.
+``hy.models.Sequence`` is the abstract base class of "iterable" Hy
+models, such as hy.Expression and hy.List.
```

Comment:
Looks like there are still a few places in the docs that aren't scoped to `hy.models`, possibly other locations as well.

Reply:
I think I forgot to check everywhere, but that should be fixed now.